language/proto/config.go

```diff
@@ -201,6 +213,10 @@ func (_ *protoLang) Configure(c *config.Config, rel string, f *rule.File) {
 				pc.ModeExplicit = true
 			case "proto_group":
 				pc.groupOption = d.Value
+			case "proto_strip_import_prefix":
+				pc.stripImportPrefix = d.Value
+			case "proto_import_prefix":
+				pc.importPrefix = d.Value
```

Comment:
You mentioned validation in another comment. I think we should validate `proto_strip_import_prefix` but not `proto_import_prefix`.

Let's require that `proto_strip_import_prefix` is absolute. It should have a leading slash, and the path should be a prefix of the package name. I don't think relative paths make sense for Gazelle: if I understand correctly, they would only be useful if your .proto files are in a subdirectory of the directory where the build file is. Gazelle won't generate rules like that.

If that's not the case, we should print an error here in `Configure`.